jina/executors/indexers/query/keyvalue.py

```diff
@@ -0,0 +1,37 @@
+from jina.executors.dump import DumpPersistor
+from jina.executors.indexers.keyvalue import BinaryPbIndexer
+from jina.executors.indexers.query import BaseQueryIndexer
+
+
+class QueryBinaryPbIndexer(BinaryPbIndexer, BaseQueryIndexer):
+    """A write-once Key-value indexer."""
```

Comment:
should we overwrite the `add`/`update`/`delete` method from the `BinaryPBIndexer` and disable them?

Reply:
Yes, that's the plan

Reply:
Done, check again if you can